2024-05-22 11:48:05,279 INFO    MainThread:32140 [wandb_setup.py:_flush():68] Configure stats pid to 32140
2024-05-22 11:48:05,279 INFO    MainThread:32140 [wandb_setup.py:_flush():68] Loading settings from C:\Users\pc\.config\wandb\settings
2024-05-22 11:48:05,279 INFO    MainThread:32140 [wandb_setup.py:_flush():68] Loading settings from C:\Users\pc\Desktop\KMU\4ÇÐ³â\Ä¸½ºÅæ\2024_Capstone\MTA_model\wandb\settings
2024-05-22 11:48:05,280 INFO    MainThread:32140 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2024-05-22 11:48:05,280 INFO    MainThread:32140 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'MTA_model\\main.py', 'program': 'C:\\Users\\pc\\Desktop\\KMU\\4ÇÐ³â\\Ä¸½ºÅæ\\2024_Capstone\\MTA_model\\main.py'}
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:_log_setup():476] Logging user logs to C:\Users\pc\Desktop\KMU\4ÇÐ³â\Ä¸½ºÅæ\2024_Capstone\MTA_model\wandb\run-20240522_114805-2oggwm8u\logs\debug.log
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:_log_setup():477] Logging internal logs to C:\Users\pc\Desktop\KMU\4ÇÐ³â\Ä¸½ºÅæ\2024_Capstone\MTA_model\wandb\run-20240522_114805-2oggwm8u\logs\debug-internal.log
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:init():516] calling init triggers
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:init():569] starting backend
2024-05-22 11:48:05,281 INFO    MainThread:32140 [wandb_init.py:init():573] setting up manager
2024-05-22 11:48:05,284 INFO    MainThread:32140 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=spawn, using: spawn
2024-05-22 11:48:05,289 INFO    MainThread:32140 [wandb_init.py:init():580] backend started and connected
2024-05-22 11:48:05,290 INFO    MainThread:32140 [wandb_init.py:init():658] updated telemetry
2024-05-22 11:48:05,353 INFO    MainThread:32140 [wandb_init.py:init():693] communicating run to backend with 60 second timeout
2024-05-22 11:48:06,257 INFO    MainThread:32140 [wandb_run.py:_on_init():2000] communicating current version
2024-05-22 11:48:06,340 INFO    MainThread:32140 [wandb_run.py:_on_init():2004] got version response upgrade_message: "wandb version 0.17.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-05-22 11:48:06,340 INFO    MainThread:32140 [wandb_init.py:init():728] starting run threads in backend
2024-05-22 11:48:11,360 WARNING MainThread:32140 [wandb_init.py:init():1099] interrupted
Traceback (most recent call last):
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 1078, in init
    run = wi.init()
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\wandb_init.py", line 739, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\interface\interface.py", line 235, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 484, in _communicate_run_start
    result = self._communicate(rec)
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\interface\interface_shared.py", line 255, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Users\pc\anaconda3\lib\site-packages\wandb\sdk\interface\router.py", line 36, in get
    is_set = self._object_ready.wait(timeout)
  File "C:\Users\pc\anaconda3\lib\threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\pc\anaconda3\lib\threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-05-22 11:48:15,479 WARNING MsgRouterThr:32140 [router.py:message_loop():77] message_loop has been closed
